sbt-launcher/src/main/scala/coursier/sbtlauncher/LauncherApp.scala

```diff
@@ -235,7 +235,7 @@ object LauncherApp extends CaseApp[LauncherOptions] {
     val appMain =
       if (params.shortCircuitSbtMain) {
         val loader = appProvider.loader()
-        val clazz = loader.loadClass("sbt.xMainImpl$")
+        val clazz = loader.loadClass("sbt.xMain$")
```

Comment:
This is only enabled (via `params.shortCircuitSbtMain`) between `1.3.0-M3` and `1.3.0-RC4`, it should be safe to keep that. Will these versions still work fine with this change?

Reply:
`1.3.0` now works in particular (since https://github.com/coursier/sbt-launcher/pull/97).

Reply:
See comment below. sbt 1.3.0 works, but not really optimally.

The shortCircuitSbtMain will not work with 1.3.0-M3 or 1.3.0-RC4 because sbt 1.3.0 broke binary compatibility in 1.3.0-RC5 by removing the xMainImpl$ class. This _could_ work if it first tries to load `xMain$` and falls back to `xMainImpl$` if it's not found.

From an sbt perspective, we will not be supporting the RC or M release so I'm not really sure of the motivation of ensuring that the coursier launcher supports them. There is really no reason for any users to not upgrade to 1.3.0 (or even better 1.3.2). It is strictly better than than the RC and M releases and users should generally not have to make any changes other than bumping the version in build.properties.